treasury/.env.sample

```diff
@@ -0,0 +1,12 @@
+REACT_APP_VERCEL_GIT_COMMIT_REF=staging # used to track which branch is used to deploy the app. 'staging' uses kovan for URLs
```

Comment:
Needs to be:
```
# used to track which branch is used to deploy the app. 'staging' uses kovan for URLs
REACT_APP_VERCEL_GIT_COMMIT_REF=staging
```

So that the example works